.github/PULL_REQUEST_TEMPLATE

```diff
@@ -7,7 +7,7 @@ Please note that the trac ticket, news fragment, and review submission portions
 ## Contributor Checklist:
 
 * [ ] The associated ticket in Trac is here: https://twistedmatrix.com/trac/ticket/<!-- Create a new one at https://twistedmatrix.com/trac/newticket and replace this comment with the ticket number. -->
-* [ ] I ran `tox -e black-reformat` to format my patch to meet the [Twisted Coding Standard](https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html)
+* [ ] I ran `tox -e lint` to format my patch to meet the [Twisted Coding Standard](https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html)
```

Comment:
an we ask people to run pre-commit  rather than lint ? or have a `tox -e precommit`

I think that we can add precomit to the dev dependency lists in setup.cfg

I think that all the black reformating should be done in a single commit and that commit added to .git-blame-ignore-revs

I would keep `tox -e lint` for CI usage ... but local devs should run pre-commit via pipx or tox or custom virtualenv

Reply:
I'm not sure what you're suggesting here

Reply:
sorry. my bad. This is ok.